npm/ng-packs/packages/core/src/lib/services/list.service.ts

```diff
@@ -150,7 +150,7 @@ export class ListService<QueryParamsType = ABP.PageQueryParams | any> implements
     this._query$.next({
       filter: this._filter || undefined,
       maxResultCount: this._maxResultCount,
-      skipCount: this._page * this._maxResultCount,
+      skipCount: this._filter ? 0 : this._page * this._maxResultCount,
```

Comment:
This line prevents the user from changing the page when the `filter` is not null. 

Case:

1. Search for something with the search input.
2. Try to change the page.
3. The page will be changed, but `skipCount` will be zero:
<img width="323" alt="Screenshot 2024-03-02 at 19 54 10" src="https://github.com/abpframework/abp/assets/61491079/f1e5a7ce-3265-4d79-ae4e-ee22f384d31c">

4. When you remove the search parameter, `skipCount` will be correct.
<img width="325" alt="Screenshot 2024-03-02 at 19 57 03" src="https://github.com/abpframework/abp/assets/61491079/a125e260-83be-45f9-8ad2-323b32d683cf">

Reply:
Hi Mustafa, i have created an issue for this problem. https://github.com/abpframework/abp/issues/19194